runelite-client/src/main/java/net/runelite/client/plugins/attackstyles/AttackStylesPlugin.java

```diff
@@ -367,6 +416,41 @@ private void hideWidget(Widget widget, boolean hidden)
 		}
 	}
 
+	private void sendChatMessage(String chatMessage)
+	{
+		if (config.warningMessages())
+		{
+			final String message = new ChatMessageBuilder()
+				.append(ChatColorType.NORMAL)
+				.append("<col=871A1A>[ATTACK STYLE]</col> ")
+				.append(chatMessage)
+				.build();
+
+			chatMessageManager.queue(
+				QueuedMessage.builder()
+					.type(ChatMessageType.GAME)
+					.runeLiteFormattedMessage(message)
+					.build());
+		}
+	}
+
+	private String returnAttackColor(Skill skill)
```

Comment:
This method name is kind of awkward... `getAttackColor()`? `attackStyleColor()`?